src/HID-Settings.h

```diff
@@ -36,7 +36,7 @@ THE SOFTWARE.
 //#define LAYOUT_FRENCH
 //#define LAYOUT_FRENCH_BELGIAN
 //#define LAYOUT_FRENCH_SWISS
-//#define LAYOUT_GERMAN
+#define LAYOUT_GERMAN
```

Comment:
The default must be US of course.

Reply:
Yep, you're right!

Reply:
Could you please fix this? Is there also a way to change the layout without editing the source files? It is a shame, that arduino still has no option to define variables like this inside the user sketch.

Reply:
As you said, it's not possible the easy way, at least according to http://forum.arduino.cc/index.php?topic=262101.0.

A constructor-based approach is possible, but also requires a rather major rewrite and could cause some overhead. I guess HID-Settings.h is fine for now?